odata4j-core/src/main/java/org/odata4j/internal/InternalUtil.java

```diff
@@ -68,6 +68,8 @@
   private static final String DATETIME_JSON_SUFFIX = ")\\/\"";
   private static final String DATETIME_JSON_PREFIX = "\"\\/Date(";
 
+  private static final String SERVER_TIME_ZONE = "GMT";

```

Comment:
I think this should be set by a parameter from OData4JLib rather than statically defined. We can default to GMT, but make it public and not final so it can be overridden. If we exception in the parser, fall back to GMT as backup.